### ISSUE TITLE
Threaded Discussions

### ISSUE THREAD

**Issue by 18alantom** [frappe/gameplan#230]
**TLDR: [Conversation Threading](https://en.wikipedia.org/wiki/Conversation_threading), e.g. what happens on HackerNews, Reddit, etc**

A _Discussion_ post (the first entry under a _Discussion_) sets the topic but the replies underneath it may be require the sub-context of some other reply, i.e. a replies to a reply.

Currently this is handled by using static blockquotes, there is no reference link to the original reply.

In long threads, or non immediate replies one has to:
- `⌘ + F` and find the parent reply.
- Scroll through the entire discussion to find the sub-context.

Without threading _Discussions_ feel like a second class feature.
